dags/burnham.py

```diff
@@ -25,24 +25,39 @@
 # metric values reported by the Glean SDK across several documents from three
 # different clients are correct.
 TEST_LABELED_COUNTER_METRICS = """
```

Comment:
Given that the dedupe logic is identical for these various queries, it may be worth extracting it to a constant, so that this becomes:

```
TEST_LABELED_COUNTER_METRICS = WITH_DISCOVERY_V1_DEDUPED + """
  SELECT
    technology_space_travel.key,
    ...
```

Reply:
That's a great idea!